terraform/bastion/Jenkinsfile-check-staging

```diff
@@ -0,0 +1,20 @@
+pipeline {
+  agent {
+    label "master"
+  }
+  stages {
+    stage("Run the bastion check for staging") {
+      steps {
+        script {
+          build(
+              job: "TDR Bastion Check",
+              parameters: [
+                  string(name: "STAGE", value: "staging"),
```

Comment:
Wondering if we should make an ongoing effort to rename all these parameters to "environment". The Jenkinsfiles are the only place where we call it a "stage"!